src/frontend/src/optimizer/plan_node/stream_group_topn.rs

```diff
@@ -111,3 +113,63 @@ impl StreamGroupTopN {
         internal_table_catalog_builder.build(dist_keys, self.base.append_only)
     }
 }
+
+impl ToStreamProst for StreamGroupTopN {
+    fn to_stream_prost_body(&self) -> ProstStreamNode {
+        use risingwave_pb::stream_plan::*;
+        let group_key = self.group_key.iter().map(|idx| *idx as u32).collect();
+
+        let group_topn_node = GroupTopNNode {
+            limit: self.limit as u64,
+            offset: self.offset as u64,
+            group_key,
+            table: Some(self.infer_internal_table_catalog().to_prost(
+                SchemaId::placeholder() as u32,
```

Comment:
```
.to_prost(
                    SchemaId::placeholder() as u32,
                    DatabaseId::placeholder() as u32,
                )
```

It seems that we are using such way everywhere. Shall we add a new function like `to_state_table_prost()` that uses these two placeholder ids by default?